src/events/acceleration.js

```diff
@@ -583,6 +568,10 @@ p5.prototype.setShakeThreshold = function(val) {
  * function draw() {
  *   fill(value);
  *   rect(25, 25, 50, 50);
+ *   describe(`50×50 black rect in center of canvas.
+ *     turns white on mobile when device turns`);
```

Comment:
This description corresponds to previous example lines 548 to 558

Reply:
Thanks!  Sending a fix...

Reply:
Fixed in 2c5e54fb